src/cockpit/389-console/src/lib/security/certificateManagement.jsx

```diff
@@ -192,44 +230,45 @@ export class CertificateManagement extends React.Component {
         });
     }
 
-    addCert () {
-        if (this.state.certName == "") {
-            this.props.addNotification(
-                "warning",
-                `Missing certificate nickname`
-            );
-            return;
-        } else if (this.state.certFile == "") {
-            this.props.addNotification(
-                "warning",
-                `Missing certificate file name`
-            );
-            return;
-        }
+    showExportModal (nickname) {
+        this.setState({
+            showExportModal: true,
+            exportNickname: nickname,
+            exportDERFormat: false,
+            exportFileName: "",
+        });
+    }
 
+    closeExportModal () {
+        this.setState({
+            showExportModal: false,
+            exportNickname: "",
+        });
+    }
+
+    exportCert () {
         this.setState({
             modalSpinning: true,
-            loading: true,
         });
         const cmd = [
             "dsconf", "-j", "ldapi://%2fvar%2frun%2fslapd-" + this.props.serverId + ".socket",
-            "security", "certificate", "add", "--name=" + this.state.certName, "--file=" + this.state.certFile
+            "security", "export-cert", this.state.exportNickname, "--output-file=" + this.state.exportFileName
         ];
-        log_cmd("addCert", "Adding server cert", cmd);
+        if (this.state.exportDERFormat) {
+            cmd += "--binary-format";
```

Comment:
It fails here because `cmd` is const. And I wonder if we'll have more `const` issues in other places as I see a lot of `const`s (not added by this PR but in general)

Reply:
Chrome Version 109.0.5414.74 (Official Build) (64-bit) here :)